frontend/coprs_frontend/coprs/logic/complex_logic.py

```diff
@@ -486,6 +486,10 @@ def generate_build_config(cls, copr, chroot_id):
                 "name": "Copr buildroot",
             })
 
+        repo_priority = copr.repo_priority or DEFAULT_COPR_REPO_PRIORITY
+        for repo in repos:
+            repo["priority"] = repo_priority
```

Comment:
I'd probably prefer just:
```python
for repo in repos:
    if copr.repo_priority in [None, DEFAULT_COPR_REPO_PRIORITY]:
        continue
    repo["priority"] = copr.repo_priority
```

Reply:
I didn't find a way how to make a field in flask-restx API model optional (discussed in today's planning) but it seems like it's possible to don't display them if the values are none via [skip_none=True](https://flask-restx.readthedocs.io/en/latest/marshalling.html#skip-fields-which-value-is-none) so I decided to set up `priority` to `None` in these cases which should hide the `99`s and `None`s
PTAL

Reply:
> I didn't find a way how to make a field in flask-restx API model optional (discussed in today's planning) but it seems like it's possible to don't display them if the values are none via [skip_none=True](https://flask-restx.readthedocs.io/en/latest/marshalling.html#skip-fields-which-value-is-none) so I decided to set up priority to None in these cases which should hide the 99s and Nones

I think we might be over-complicating things. When use this PR and remove the `skip_none` thing

```diff
- @apiv3_project_chroots_ns.marshal_with(project_chroot_build_config_model, skip_none=True)
+ @apiv3_project_chroots_ns.marshal_with(project_chroot_build_config_model)
```

It seems to work anyway and when the priority is not set, its value is `null`.

![Screenshot_2023-07-31_23-12-46](https://github.com/fedora-copr/copr/assets/2151771/9a84f0d0-8219-4419-887c-cd899397e353)

This is IMHO a behavior that we want.